documentation/documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/python_evaluator.md

```diff
@@ -1,13 +1,13 @@
 # Python evaluator
-## General usage
-This evaluator is responsible for running and executing tests on a student's Python code.
+## Algemeen gebruik
+Deze evaluator is verantwoordelijk voor het uitvoeren en testen van de Python-code van een student.
 
-## Structure
-When submitting the project a teacher can add a requirements manifest `req-manifest.txt`, this way only the packages in the requirements file are usable on the evaluator.
+## Structuur
+Bij het indienen van het project kan een leraar een vereisten toevoegen via het bestand `req-manifest.txt`. Op deze manier zijn alleen de pakketten in het vereistenbestand bruikbaar op de evaluator.
 
-When no manifest is present, students are able to install their own depedencies with a `requirements.txt` and a `dev-requirements.txt`.
-Or the teacher can add a `requirements.txt` if they want to pre install dependencies that a are present for testing the project.
+Wanneer er geen manifest aanwezig is, kunnen studenten hun eigen paketten installeren met een `requirements.txt` en een `dev-requirements.txt`.
+Of de leraar kan een `requirements.txt` toevoegen als ze paketten vooral willen installeren die aanwezig moeten zijn voor het testen van het project.
```

Comment:
This is a weird sentence, did you mean vooraf instead of vooral?

Reply:
https://github.com/SELab-2/UGent-3/pull/333/commits/ea486be4a86cf18a35b56113e6964658bfac9b71